src/elf/reloc.rs

```diff
@@ -253,16 +253,58 @@ pub mod reloc64 {
     pub const SIZEOF_RELA: usize = 8 + 8 + 8;
     pub const SIZEOF_REL: usize = 8 + 8;
 
+    #[cfg(not(all(
```

Comment:
Why would these bit masking helper functions be cfg’d out on mips systems?

Reply:
Ah I see further down is the other condition. So I have to reread the linked thread but is the issue with how mips itself does some bit masking here or if the binary is from a mips arch ? If the latter a cfg is not appropriate as we should be able to read a mips elf binary on a whatever machine.